academic/ViennaRNA/ViennaRNA.SlackBuild

```diff
@@ -101,10 +107,7 @@ find -L . \
 # for Z-score filtering via statically linked libsvm:
 # - SVM Z-score filter in RNALfold
 # - GNU Scientific Library for RNApvmin
-# RNAxplorer blocks configuration despite intalling dependencies lapacke and lapack, so ignore it
-# https://github.com/ViennaRNA/ViennaRNA/issues/206
-rnaxplorer=""    ; [ "${RNAXPLORER:-no}" != "yes" ]     && rnaxplorer="--without-rnaxplorer"
-# everything else is enabled
+# everything is enabled here
 cluster=""       ; [ "${CLUSTER:-yes}" != "no" ]        && cluster="--with-cluster"
 kinwalker=""     ; [ "${KINWALKER:-yes}" != "no" ]      && kinwalker="--with-kinwalker"
 # Default enabled features:
```

Comment:
I think there is also a mistake there 2 lines down:

```
kinfold=""       ; [ "${KINFOLD:-yes}" != "no" ]        && kinwalk="--with-kinfold"
```

that `&& kinwalk...` the variable `kinwalk` is never used. Shouldn't it be `kinfold`?

Reply:
Latest :+1: